commons/src/main/scala/za/co/absa/spline/common/config/UpperCaseEnvironmentConfiguration.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.common.config
+
+import org.apache.commons.configuration.EnvironmentConfiguration
+import org.apache.commons.lang.StringUtils
+
+class UpperCaseEnvironmentConfiguration extends EnvironmentConfiguration {
```

Comment:
```suggestion
class NotationInsensitiveEnvironmentConfiguration extends EnvironmentConfiguration {
```

Reply:
I have even better idea - spit the responsibility. I can make this class only doing UPPER_SNAKE_NOTATION, and instead of calling the `super` method for the fallback add another `EnvironmentConfiguration` instance to the default config resolver stack

Reply:
`UPPER_SNAKE_NOTATION` interesting 🐍 😄